content/geometry/HalfPlane.h

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * Author: chilli
+ * Date: 2019-05-05
+ * License: CC0
+ * Source: https://github.com/zimpha/algorithmic-library/blob/master/computational-geometry/polygon.cc
+ * Description: Computes the intersection of a set of half-planes.
```

Comment:
Can you explain this better? What format is the input and output in?

Reply:
I'm not really sure how to describe the input... I wrote "Input is given as a set of planes, facing left.", but that seems like a pretty poor description.